expr/parse.go

```diff
@@ -616,7 +613,7 @@ func (t *tree) v(depth int) Node {
 	case lex.TokenNull:
 		t.Next()
 		return NewNull(cur)
-	case lex.TokenStar:
+	case lex.TokenStar, lex.TokenMultiply:
```

Comment:
@mattayes the rest of the parser looks for either `TokenStar` _and_ `TokenMultiplier`. 

A filter ql statement like this fails parsing: `FILTER and ( INCLUDE akci28akchek4nakc, country = "Germany")` when `akci28akchek4nakc` is the id for the "all" segment. After the include clause is resolved, the ql will be `FILTER and ( *, country = "Germany")`. The parser errors on the `*` token because the code only checks for `TokenStar`, not `TokenMultiplier`. We could change the lexer, but that would have a much larger blast radius. The effects of changing the parser are much more simple to determine.

Reply:
@LinaShadrach It's simpler yes, however it's just masking/working around a bug instead of solving the underlying problem: why do we have different tokens for the same string (`*`)? AFAIK that isn't something a lexer should do and it's up to the parser to add semantic meaning to the tokens.

Reply:
@mattayes i am not against altering the lexer, but i think that will take significant time and testing. I think it'd be good to fix the issue for the customer sooner than I could deliver that chunk of work.

Reply:
@LinaShadrach That sounds good, can we add a comment about changing this and a regression test to ensure Multiply always behaves the same as `Star`?

Reply:
Ah, I had some reservations/uncertainty about this when I started looking yesterday, but this thread is helpful.